consultation_analyser/support_console/urls.py

```diff
@@ -5,4 +5,5 @@
 urlpatterns = [
     path("", views.support_home),
     path("sign-out/", views.sign_out),
+    path("ml-pipeline-test/", views.ml_pipeline_test),
```

Comment:
So I think we could do this in a more RESTful way.

Eg

- have a /consultations page listing and linking to consultations
- have a /consultation/slug page
- on that page have a "run pipeline" button

Reply:
Now updated